bot/exts/evergreen/latex.py

```diff
@@ -48,7 +47,7 @@ def _render(text: str) -> Union[BytesIO, str]:
             return rendered_image
 
         except ValueError as e:
-            return str(e)
+            raise commands.BadArgument(str(e))
```

Comment:
I'm not sure if we want to output the full traceback here. What does `e` look like in this case?

Reply:
![image](https://user-images.githubusercontent.com/69471777/114189916-5c5a4380-9968-11eb-8344-58af6a35d67c.png)

Reply:
reasonable imo

Reply:
Yea, that's fine then. Wasn't sure if it would have a huge traceback.